### PR TITLE
Add new notebook

### PR DESCRIPTION
**Pull request by shivdeep-singh-ibm**

### CODE REVIEW

**Review by Bytes-Explorer — COMMENTED** [review]
Descriptions are missing, need to be added.

**Comment by Bytes-Explorer** [comment]
Parameters in the first cell should be hidden. Only user facing params should be retained and adequate description should be added.

**Review by Bytes-Explorer — COMMENTED** [review]
Please make them as a single pip install that Maroun is building.

**Comment by Bytes-Explorer** [comment]
All cells should have a markdown format on what are the parameters, how should an end user use them etc, See https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.FeatureHasher.html#sklearn.feature_extraction.FeatureHasher for inspiration.

**Comment by Bytes-Explorer** [comment]
Change ray_params to an easy to use name for a data scientist

File: examples/notebooks/code/transform_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Data Prep Kit
+git+https://github.com/IBM/data-prep-kit.git#egg=data_prep_toolkit&subdirectory=data-processing-lib/python
+git+https://github.com/IBM/data-prep-kit.git#egg=data_prep_toolkit_ray&subdirectory=data-processing-lib/ray
+
+# Ededup Transform
+git+https://github.com/IBM/data-prep-kit.git#egg=dpk_ededup_transform_python&subdirectory=transforms/universal/ededup/python
+git+https://github.com/IBM/data-prep-kit.git#egg=dpk_ededup_transform_ray&subdirectory=transforms/universal/ededup/ray
+
+# Tokenisation
```
Comment (santoshborse) [inline examples/notebooks/code/transform_requirements.txt]: spell check - should be `Tokenization`

File: examples/notebooks/code/sample-notebook.ipynb [hunk 2]
```diff
@@ -0,0 +1,1068 @@
...
+    "\n",
+    "The following transformations are applied in order.\n",
+    "- HF2Parquet\n",
+    "- Exact Dedup\n",
+    "- Doc Id\n",
+    "- Fuzzy Dedup\n",
+    "- Prog Lang Select\n",
+    "- Filtering\n",
+    "- Repo Level Grouping\n",
+    "- Tokenisation\n",
```
Comment (santoshborse) [inline examples/notebooks/code/sample-notebook.ipynb]: spell check - should be `Tokenization`

File: examples/notebooks/code/sample-notebook.ipynb [hunk 3]
```diff
@@ -0,0 +1,1068 @@
...
+    "    launcher.launch()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "byK75Kb1A3E7"
+   },
+   "source": [
+    "## Tokenisation"
```
Comment (santoshborse) [inline examples/notebooks/code/sample-notebook.ipynb]: spell check - should be `Tokenization`

**Review by santoshborse — DISMISSED** [review]
`Tokenization` spelled wrong in multiple places

**Review by touma-I — APPROVED** [review]

**Comment by touma-I** [comment]
This iteration meets current needs. Expect more iterations in new PR

**Pull request merged**